chainstate/src/detail/tests/double_spend_tests.rs

```diff
@@ -15,231 +15,139 @@
 //
 // Author(s): A. Sinitsyn
 
-use crate::detail::tests::*;
+use crate::detail::{spend_cache::error::StateUpdateError, tests::*};
 use chainstate_storage::BlockchainStorageRead;
-use common::chain::block::timestamp::BlockTimestamp;
-use common::chain::block::{Block, ConsensusData};
-use common::chain::{OutPointSourceId, Transaction, TxInput, TxOutput};
-use common::primitives::{time, Amount, Id};
-
+use common::{
+    chain::{
+        block::{timestamp::BlockTimestamp, Block, ConsensusData},
+        OutPointSourceId, Spender, Transaction, TxInput, TxOutput,
+    },
+    primitives::{time, Amount, Id},
+};
+use crypto::random::{self, Rng};
+
+// Process a block where the second transaction uses the first one as input.
```

Comment:
Isn't it idiomatic to use /// instead of // for documentation? Or it doesn't matter?

Reply:
Usually I prefer to use documentation comments even for private functions (a documentation can still be generated for them), but not for tests. I guess I don't have solid reasons for that, so I can change these comments if you like.

Reply:
Not really... but I was wondering since you went for documentation above the function why it's not done "the rust way"'

Reply:
I have checked: the documentation isn't generated for tests even with the `--document-private-items` flag. It can be worked around by replacing `#[cfg(test)]` with `#[cfg(any(test, doctest))]`, but I doubt that we need that.